app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharecterView.kt

```diff
@@ -1,53 +1,71 @@
 package com.puzzlebench.clean_marvel_kotlin.presentation.mvp
 
-import android.support.v7.widget.GridLayoutManager
+import android.os.Bundle
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
 import android.view.View
 import com.puzzlebench.clean_marvel_kotlin.R
 import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
 import com.puzzlebench.clean_marvel_kotlin.presentation.MainActivity
 import com.puzzlebench.clean_marvel_kotlin.presentation.adapter.CharacterAdapter
 import com.puzzlebench.clean_marvel_kotlin.presentation.extension.showToast
-import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.activity_main.recycleView
+import kotlinx.android.synthetic.main.activity_main.progressBar
 import java.lang.ref.WeakReference
 
-class CharecterView(activity: MainActivity) {
+
+class CharacterView(activity: MainActivity) {
+
+    companion object {
+        const val EXTRA_CHARACTER = "character"
+        private const val TAG = "txn_tag"
+    }
```

Comment:
companion object should be at the bottom of the class

Reply:
Done